src/cmd/test.ts

```diff
@@ -14,11 +14,15 @@ import { validateValues } from './validate-values'
 
 const cmdName = getFilename(import.meta.url)
 const tmpFile = '/tmp/otomi/test.yaml'
-const debug: OtomiDebugger = terminal(cmdName)
+let debug: OtomiDebugger = terminal(cmdName)
 
 const cleanup = (argv: Arguments): void => {
+  debug = terminal(cmdName)
   if (argv.skipCleanup) return
-  unlinkSync(tmpFile)
+  try {
+    unlinkSync(tmpFile)
+  }
+  catch (e) { debug.error(e) }
```

Comment:
Instead of a try catch:
```
if(existsSync(tmpFile)) unlinkSync(tmpFile)
```

Then no try/catch is necessary

Reply:
Sort of, this is the implementation of existsSync:

```
fs.existsSync = function(path) {
  try {
    handleError((path = getPathFromURL(path)));
    nullCheck(path);
    binding.stat(pathModule._makeLong(path));
    return true;
  } catch (e) {
    return false;
  }
};
```

But I agree that this would be the preferable solution.

Reply:
I understand, but as @Morriz stated, prefer not to do try/catch. So if we know it exists, then we can safely remove it

Reply:
Apparently you do not? Your suggestion of existsSync is basically a wrapper of try catch right?